golang/x/relay/types/validator.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+type FilledRequestInfo struct {
+	InputIndex  uint8     `json:"inputIndex"`
+	OutputIndex uint8     `json:"outputIndex"`
+	ID          RequestID `json:"id"`
+}
+
+type FilledRequests struct {
+	Proof    SPVProof            `json:"proof"`
+	Requests []FilledRequestInfo `json:"requests"`
```

Comment:
Can we call this `Filled` so that it's clear it's something other than a `[]Request`?